_i18n/en.yml

```diff
@@ -331,6 +331,7 @@ faq:
   a1: Monero has value because people are willing to buy it. If no one is willing to buy Monero, then it will not have any value. Monero’s price increases if demand exceeds supply, and it decreases if supply exceeds demand.
   q2: How can I get Monero?
   a2: You can buy Monero from an exchange or from an individual. Alternatively, you can try mining Monero to get coins from the @block reward.
+  a2exchanges: "In past, you needed Bitcoin to buy Monero, but that's not the case anymore. You can directly trade Monero for national currencies (USD, EUR, GBP, etc) or other cryptocurrencies on many exchanges. Some require KYC (proof of identification), others do not, like decentralized exchanges. On this website is available a list of services, which includes exchanges (both centralized and decentralized), where it's possible to buy/sell Monero (XMR):"
```

Comment:
Some require KYC (proof of identification); others do not, ...     (use semicolon)

Here is available a list of exchanges where it's possible to buy/sell Monero (XMR):           (simplify)

Reply:
Added semicolon and changed last phrase into `On this website is available a list of exchanges where it's possible to buy/sell Monero (XMR):`. I used "On this website" instead of "here".